projects/1.16/assets/biomancy/biomancy/lang/zh_cn.json

```
@@ -0,0 +1,265 @@
+  "item.biomancy.thick_wool_sheep_spawn_egg": "厚毛羊刷怪蛋",
+  "item.biomancy.nutrient_slurry_cow_spawn_egg": "营养奶牛刷怪蛋",
+
+  "fluid.minecraft.water": "水",
+  "fluid.biomancy.nutrient_slurry": "营养液",
+
+  "block.biomancy.meat_berry_bush": "肉莓丛",
+  "block.biomancy.flesh_tentacle": "血肉触手",
+
+  "block.biomancy.flesh_block": "血肉块",
+  "block.biomancy.flesh_block_slab": "血肉台阶",
+  "block.biomancy.flesh_block_stairs": "血肉楼梯",
+  "block.biomancy.necrotic_flesh_block": "坏死肉块",
+  "block.biomancy.fleshborn_door": "血肉门",
+  "block.biomancy.fleshborn_trapdoor": "血肉活板门",
+  "block.biomancy.fleshborn_pressure_plate": "血肉压力板",
+  "block.biomancy.fleshborn_chest": "血肉箱",
+  "block.biomancy.gulge": "深邃胃囊",
+  "block.biomancy.chewer": "咀嚼器",
+  "block.biomancy.digester": "消化器",
+  "block.biomancy.solidifier": "分泌腺",
+  "block.biomancy.decomposer": "分解器",
+  "block.biomancy.meatsoup_cauldron": "血肉池",
+  "block.biomancy.evolution_pool": "进化池",
+
+  "container.biomancy.gulge": "深邃胃囊",
+  "container.biomancy.fleshborn_chest": "血肉箱",
+  "container.biomancy.chewer": "咀嚼器",
+  "container.biomancy.digester": "消化器",
+  "container.biomancy.solidifier": "分泌腺",
+  "container.biomancy.decomposer": "分解器",
+  "container.biomancy.evolution_pool": "进化池",
+
+  "entity.biomancy.flesh_blob": "肉块",
+  "entity.biomancy.oculus_observer": "观察之眼",
+  "entity.biomancy.fleshkin": "血肉幼体",
+  "entity.biomancy.boomling": "爆虫",
+  "entity.biomancy.failed_sheep": "失败品羊",
+  "entity.biomancy.chroma_sheep": "缤纷羊",
+  "entity.biomancy.silky_wool_sheep": "丝毛羊",
+  "entity.biomancy.thick_wool_sheep": "厚毛羊",
+  "entity.biomancy.failed_cow": "失败品牛",
+  "entity.biomancy.nutrient_slurry_cow": "营养奶牛",
+  "entity.biomancy.brood_mother": "巢穴之母",
+
+  "enchantment.biomancy.climbing": "千足虫",
+  "enchantment.biomancy.bullet_jump": "跳蚤",
+  "enchantment.biomancy.attuned_bane": "共鸣克星",
+  "enchantment.biomancy.attuned_bane_of": "%1$s 的克星",
+  "enchantment.biomancy.quick_shot": "射速",
+  "enchantment.biomancy.max_ammo": "扩容",
+
+  "effect.biomancy.dread": "恐惧",
+  "effect.biomancy.ravenous_hunger": "极度饥饿",
+  "effect.biomancy.flesh_eating_disease": "坏死",
+
+  "reagent.biomancy.growth_serum": "生长血清",
+  "reagent.biomancy.rejuvenation_serum": "复原血清",
+  "reagent.biomancy.breeding_stimulant": "育种剂",
+  "reagent.biomancy.absorption_boost": "伤害吸收剂",
+  "reagent.biomancy.insomnia_cure": "失眠治疗剂",
+  "reagent.biomancy.cleansing_serum": "净化血清",
+  "reagent.biomancy.decay_agent": "坏死血清",
+  "reagent.biomancy.mutagen_serum": "诱变血清",
+  "reagent.biomancy.blood_sample": "血液样本",
+
+  "attribute.generic.attack_distance": "攻击距离",
+
+  "tooltip.biomancy.unveiling_oculi": "赋予你细致的视线，揭示未曾所见之物。你不会再被黑暗或魔法致盲。",
+  "tooltip.biomancy.fleshborn_war_axe": "一把暴脾气的斧头，当持有者的生命值低于敌人最大生命值的 75% 时，有 60% 的几率造成敌人最大生命值的 30% 的伤害。",
+  "tooltip.biomancy.fleshborn_khopesh": "一把由斧头进化而来的镰刀剑。",
+  "tooltip.biomancy.blood_curse_axe": "持有者生命值越低，攻击速度越快的斧头。",
+  "tooltip.biomancy.leech_claw": "攻击会回复造成伤害 50% 的生命。\n还会在重击时获得 15 秒的额外生命，最多可叠加 5 层。",

```

|Feedback Priority|
|---|
|🧗‍♀️ Boulder / **Blocking**|

```suggestion
  "tooltip.biomancy.leech_claw": "攻击会恢复造成伤害 50% 的生命值。\n还会在暴击时获得 15 秒的伤害吸收效果，效果强度最多叠加 5 层。",
```
伤害吸收 (Absorption) - [MC]我的世界原版 (Minecraft) - MC百科|最大的Minecraft中文MOD百科
https://www.mcmod.cn/item/9399.html

伤害 - Minecraft Wiki，最详细的官方我的世界百科
https://minecraft.fandom.com/zh/wiki/%E4%BC%A4%E5%AE%B3#.E6.9A.B4.E5.87.BB

回复虽有恢复之意，但为何不直接使用恢复？